integreat_compass/cms/views/offers/my_offers_list_view.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from django.contrib import messages
+from django.db import transaction
+from django.shortcuts import redirect, render
+from django.utils.decorators import method_decorator
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...decorators import permission_required
+from ...models import Offer
+
+logger = logging.getLogger(__name__)
+
+
+@method_decorator(permission_required("cms.change_offer"), name="dispatch")
+class MyOffersListView(TemplateView):
+    """
+    View showing all of a user's offers
+    """
+
+    template_name = "offers/my_offers_list.html"
+
+    def get(self, request, *args, **kwargs):
```

Comment:
If you're only adding context and don't change the rendering process, it makes more sense to override `get_context_data()` instead of `get()`.